discord/abc/discordobject.py

```diff
@@ -1,19 +1,29 @@
 from __future__ import annotations
-
-from datetime import datetime
+from typing import TYPE_CHECKING
 
 from pydantic import BaseModel
 
 from ..types.snowflake import Snowflake
 
 
-class DiscordObject(BaseModel):
+if TYPE_CHECKING:
+    from discord import Client
+
 
+class DiscordObject(BaseModel):
     id: Snowflake
-    created_at: datetime
```

Comment:
Why was `created_at` removed? It's basically what all snowflake models have.

Reply:
`created_at` is added to `Snowflake`